packages/ui/resolution/tests/resolution-specs.js

```diff
@@ -13,25 +14,205 @@ describe('Resolution Specs', () => {
 		{name: 'hd', pxPerRem: 16, width: 1280, height: 720, aspectRatioName: 'hdtv'},
 		{name: 'fhd', pxPerRem: 24, width: 1920, height: 1080, aspectRatioName: 'hdtv', base: true},
 		{name: 'uw-uxga', pxPerRem: 24, width: 2560, height: 1080, aspectRatioName: 'cinema'},
-		{name: 'uhd', pxPerRem: 48, width: 3840, height: 2160, aspectRatioName: 'hdtv'}
+		{name: 'qhd', pxPerRem: 32, width: 2560, height: 1440, aspectRatioName: 'hdtv'},
+		{name: 'wqhd', pxPerRem: 32, width: 3440, height: 1440, aspectRatioName: 'cinema'},
+		{name: 'uhd', pxPerRem: 48, width: 3840, height: 2160, aspectRatioName: 'hdtv'},
+		{name: 'uhd2', pxPerRem: 96, width: 7680, height: 4320, aspectRatioName: 'hdtv'}
 	];
+	const VGA = {height: 480, width: 640};
+	const XGA = {height: 768, width: 1024};
```

Comment:
In screenTypes, types starting from hd are used. Is there a reason why 'vga' and 'xga' were added in this test? Is it because you can test smaller cases?

Reply:
I added unit tests based on existing unit test in `resolution-specs.js`.
Because VGA and XGA have been in screenType list, I think it's better to keep them.